plugins/sentinelone/unit_test/test_quarantine_multiple.py

```diff
@@ -0,0 +1,29 @@
+import os
+import sys
+
+sys.path.append(os.path.abspath("../"))
+
+from unittest import TestCase
+from unittest.mock import patch, Mock
+
+from komand_sentinelone.actions.quarantine_multiple import QuarantineMultiple
+from komand_sentinelone.actions.quarantine_multiple.schema import Input, Output
+
+from util import Util
+
+
+class TestQuarantineMultiple(TestCase):
+    @classmethod
+    @patch("requests.post", side_effect=Util.mocked_requests_get)
+    def setUpClass(cls, mock_request: Mock) -> None:
+        cls.action = Util.default_connector(QuarantineMultiple())
+
+    @patch("requests.request", side_effect=Util.mocked_requests_get)
+    @patch("requests.get", side_effect=Util.mocked_requests_get)
+    def test_should_success(self, mock_request: Mock, mock_get: Mock) -> None:
+        expected = {
+            Output.SUCCESSFUL: ["hostname123"],
```

Comment:
As this is to check a multiple quarantine action, can you add a test to check for multiple as well as one hostname?